src/reporting/market.se

```diff
@@ -211,7 +215,6 @@ def tryFinalizeAutomatedReport():
         return(0)
     self.finalPayoutDistributionHash = self.automatedReporterPayoutDistributionHash
     self.finalizationTime = block.timestamp
-    self.privateRewardBondHolders()
     self.reportingWindow.updateMarketPhase()
```

Comment:
While it is correct that we don't need to reward bond holders here, we _do_ need to transfer their REP to the winning reporting token _IF_ the dispute was incorrect.

Something along the lines of this:
```
if finalPayoutDistributionHash == bondToken.getPayoutDistributionHash():
    amount = reputationToken.balanceOf(bondToken)
    reputationToken.transferFrom(bondToken, self.getFinalWinningReportingToken(), amount)
```

Same for the other 3 similar functions.